test/fulfillment_order_test.rb

```diff
@@ -132,5 +132,173 @@ def setup
         assert_equal 'closed', fulfillment_order.status
       end
     end
+
+    context "#fulfillment_request" do
+      should "make a fulfillment request for a fulfillment order" do
+        original_fulfillment_order = ActiveSupport::JSON.decode(load_fixture('fulfillment_order'))
+        submitted_fulfillment_order = original_fulfillment_order.clone
+        submitted_fulfillment_order['id'] = 2
+        submitted_fulfillment_order['status'] = 'open'
+        submitted_fulfillment_order['request_status'] = 'submitted'
+        unsubmitted_fulfillment_order = original_fulfillment_order.clone
+        unsubmitted_fulfillment_order['id'] = 3
+        unsubmitted_fulfillment_order['request_status'] = 'unsubmitted'
+        original_fulfillment_order['status'] = 'in_progress'
```

Comment:
It looks like you're describing a split fulfillment request with this test. Ex.
FO for 3 hats
Request 1 hat, new FO created with 1 hat (open/submitted), remaining FO created with 2 hats (open/unsubmitted).

In this case, the original FO should be `closed`.